openedx/features/caliper_tracking/tests/expected/edx.course.student_notes.added.json

```diff
@@ -0,0 +1,44 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Bookmarked",
```

Comment:
I don't believe this is a `Bookmarked` event. Please find another appropriate action for this.

Reply:
Done